handsontable/src/core/hooks/constants.js

```diff
@@ -314,6 +314,17 @@ export const REGISTERED_HOOKS = [
    */
   'beforeCreateCol',
 
+  /**
+   * Fired after the cache of the column sequence has been updated.
+   *
+   * @event Hooks#afterColumnSequenceCacheUpdate
```

Comment:
```suggestion
   * Fired after the cache of the column sequence has been updated.
   *
   * @since 16.2.0
   * @event Hooks#afterColumnSequenceCacheUpdate
```